docs/knowledge_base/analyzing-dependencies.md

```diff
@@ -22,7 +22,7 @@ The Phylum CLI supports processing many different lockfiles:
 | `go`          | `go.sum`                                                               |
 | `cargo`       | `Cargo.lock`                                                           |
 | `spdx`        | `*.spdx.json` <br /> `*.spdx.yaml` <br /> `*.spdx.yml` <br /> `*.spdx` |
-| `cyclonedx`   | `bom.json` <br /> `bom.xml`                                            |
+| `cyclonedx`   | `bom.json` <br /> `*.bom.json` <br /> `bom.xml`  <br /> `*.bom.xml`    |
```

Comment:
```suggestion
| `cyclonedx`   | `*bom.json` <br /> `*bom.xml`                                          |
```

I had this exact change sitting in my uncommitted changes...waiting for a PR to put it in. The reason for the suggested change is that the current logic for CycloneDX lockfile detection does not include a check for the `.` preceding `bom.json` or `bom.xml`:

```
    fn is_path_lockfile(&self, path: &Path) -> bool {
        path.file_name()
            .and_then(OsStr::to_str)
            .map_or(false, |name| name.ends_with("bom.json") || name.ends_with("bom.xml"))
    }
```

The suggestion also simplifies the glob syntax, which is still valid for representing _both_ files that are _only_ `bom.json` and those that _end with_ `bom.json`:

```
❯ ls -alh tests/fixtures/*bom.json
-rw-r--r--  1 maxrake  staff    87K Aug 22 15:10 tests/fixtures/bom.json
-rw-r--r--  1 maxrake  staff   1.0K Aug 22 15:10 tests/fixtures/nested_bom.json

❯ ls -alh tests/fixtures/*bom.xml
-rw-r--r--  1 maxrake  staff    86K Aug 22 15:10 tests/fixtures/bom.xml
```